src/lib.rs

```diff
@@ -5,8 +5,11 @@
     rustc::internal
 )]
 #![deny(missing_docs)]
-
-//! A crate to run the Rust compiler (or other binaries) and test their command line output.
+#![cfg_attr(doc, doc = include_str!("../README.md"))]
+#![cfg_attr(
+    not(doc),
```

Comment:
Hmm.. is this the intended usage? Why not just unconditionally import? Or make `doc` attributes disappear without cfg(doc) in general?

Reply:
I think it does at later stages. But if it made them disappear right after the lint pass, rustdoc wouldn't have access to them.

I didn't unconditionally import to prevent adding a watcher over the readme file because I know some people are sensible to these kind of things but if it doesn't matter for you, I can do it. It'll simplify the code too. :)